packages/plugin-ext/src/main/browser/style/plugin-sidebar.css

```diff
@@ -35,6 +35,13 @@
     border-bottom: 1px solid;
 }
 
+.theia-side-panel .theia-plugins .pluginHeaderContainer {
+    padding-left: 20px;
+}
+
+
+ . 
```

Comment:
I think there is a minor leftover in the file, the extra lines and period :(

Reply:
Oups, thanks!


.